pkg/cmd/roachtest-selector/README.md

```diff
@@ -0,0 +1,7 @@
+# Documentation
+
+This is the roachtest selector for selecting tests to run based on the test criteria.
```

Comment:
The README should at minimum explain how this is supposed to work end-to-end, preferably with examples. The referenced wiki is also pretty sparse on the actual details.

Reply:
I considered the doc to be the go to place. I actually added details on how the system works in the doc. I will add more details here.